### ISSUE TITLE
Accessibility-related ideas

### ISSUE THREAD

**Issue by davidtheclark** [davidtheclark/unfinishedToggler#11]
- Next/prev actions should be accessible even without next/prev buttons. That way, for example, on an ARIA-enabled `tablist` we can assign next/prev actions to keydown events.
- An `onInit` callback would be useful for triggering a function that adds all kinds of ARIA-attributes.
- Try to make it the default to leave UFT classes out of the markup and only add them programmatically. Option should indicate whether any specific group should start open, otherwise they all start closed.
- More plugins could be accessibility plugins for specific components, adding functionality and ARIA features.
